frontend/views/reports.js

```diff
@@ -56,6 +56,10 @@ var ReportTableView = Backbone.View.extend({
             context.rows = this.results.rows;
             this.chartEl.show();
             this.chartEl.plot(this.results.chartData, this.results.chartOptions);
+            var self = this;
+            window.onresize = function () {
+                self.chartEl.plot(self.results.chartData, self.results.chartOptions);
```

Comment:
Two thoughts:

* Let's use http://api.jquery.com/resize/ to catch resize events. That way we shouldn't interfere with any other resize event handlers that might be around.

* That doc warns that in some browsers, resize events fire continuously during a resize. We should wait until they've stopped before doing something expensive like replotting our chart. There are examples in map.js (bounds_changed, center_changed) of setting a timer in a handler and not doing the work until the timer expires without the handler firing again.